src/main/java/frc/robot/RobotContainer.java

```diff
@@ -271,19 +322,84 @@ private void configureButtonBindings() {
     final Runnable resetGyro =
         Constants.currentMode == Constants.Mode.SIM
             ? () -> drive.setPose(driveSimulation.getSimulatedDriveTrainPose())
-            : () -> drive.setPose(new Pose2d(drive.getPose().getTranslation(), new Rotation2d()));
+            : () ->
+                drive.setPose(
+                    new Pose2d(drive.getPose().getTranslation(), new Rotation2d())); // zero gyro
 
     // Default command, normal field-relative drive
     drive.setDefaultCommand(
         DriveCommands.joystickDrive(
             drive,
-            () -> OI.getAxisSupplier(OI.Driver.LeftY).get(),
-            () -> OI.getAxisSupplier(OI.Driver.LeftX).get(),
-            () -> OI.getAxisSupplier(OI.Driver.RightX).get()));
+            OI.getAxisSupplier(OI.Driver.LeftY),
+            OI.getAxisSupplier(OI.Driver.LeftX),
+            OI.getButton(Driver.RSB).getAsBoolean()
+                ? () -> 0.0
+                : OI.getAxisSupplier(OI.Driver.RightX)));
     OI.getButton(OI.Driver.Back).onTrue(Commands.runOnce(resetGyro, drive).ignoringDisable(true));
+    OI.getButton(OI.Driver.RSB)
+        .whileTrue(
+            DriveCommands.AlignToReef(
+                drive,
+                OI.getAxisSupplier(Driver.LeftY),
+                OI.getAxisSupplier(Driver.LeftX),
+                drive.getAlignRotation()));
+
+    /* This is for creating the button mappings for logging what coral have been scored
+     * The Driverstation has a hard limit of 32 buttons so we use 2 different vjoy controllers
+     * to get the effective 64 buttons that we need for logging. this first 16 buttons of every controller are
+     * used for the front and back coral scored poses. */
+    int rows = 3;
+    for (int i = 0; i < Constants.kPoleLetters.length / 2; i++) {
+      for (int j = 0; j < rows; j++) {
+        OI.getButton(OI.StreamDeck.streamDeckButtons[0][i * rows + j])
+            .onTrue(drive.setPoseScored(Constants.kPoleLetters[i], j))
+            .onFalse(drive.setPoseScored(Constants.kPoleLetters[i], j));
+        OI.getButton(OI.StreamDeck.streamDeckButtons[1][i * rows + j])
+            .onTrue(drive.setPoseScored(Constants.kPoleLetters[i + 6], j))
+            .onFalse(drive.setPoseScored(Constants.kPoleLetters[i + 6], j));
+      }
+    }
 
-    OI.getPOVButton(OI.Operator.DPAD_LEFT).whileTrue(intake.pivotUpCommand());
-    OI.getPOVButton(OI.Operator.DPAD_RIGHT).whileTrue(intake.pivotDownCommand());
+    // Button to update Setpoints of the elevator based on the Stream Deck nobs
+    OI.getButton(OI.StreamDeck.streamDeckButtons[1][31])
+        .onTrue(
+            elevator
+                .tuneSetpoints(
+                    OI.getAxisSupplier(OI.StreamDeck.Nob1),
+                    () -> DriverStation.getStickAxis(2, 1),
```

Comment:
Why is this one different from the others?

Reply:
Left over from when me and Kian were trying to figure out why the axis wasn't changing. We still have not fixed the issue, I put a TODO there at the very least to remind us of this.